packages/core/src/tools/builders/BlockToolBuilder.ts

```diff
@@ -0,0 +1,217 @@
+import type { BlockToolAdapter } from "@editorjs/dom-adapters";
+import { BaseToolBuilder, InternalBlockToolSettings, UserSettings } from './BaseToolBuilder.js';
+import type {
+  BlockAPI,
+  BlockToolData,
+  ConversionConfig,
+  PasteConfig, SanitizerConfig, ToolboxConfig,
+  ToolboxConfigEntry
+} from '@editorjs/editorjs';
+import { isEmpty, cacheable, isObject } from '@editorjs/helpers';
+import { type InlineToolBuilder } from './InlineToolBuilder.js';
+import { ToolType } from './ToolType.js';
+import { type BlockTuneBuilder } from './BlockTuneBuilder.js';
+import { ToolsCollection } from './ToolsCollection.js';
+import { BlockToolConstructor as BlockToolConstructable, BlockTool as IBlockTool } from '../../entities/BlockTool.js';
+
+/**
+ * Class to work with Block tools constructables
+ */
+export class BlockToolBuilder extends BaseToolBuilder<ToolType.Block, IBlockTool> {
```

Comment:
we have `api.tools.getBlockTools()` method that exposes builders (facades), so they probably should have interfaces in `sdk`